src/lib/hooks/useAnimateSplash.js

```diff
@@ -0,0 +1,31 @@
+import { useCallback, useEffect } from 'react'
```

Comment:
as this logic is used in 3 places we have to concentrate it a single place. for example in the following hook
(you're free to remove this file if you'll select another way, for example simple service like userStorage or functions set like analytics)